### PR TITLE
Add data community section to gov.uk developer docs

### PR DESCRIPTION
**Pull request by jonathanglassman**
To centralise the newly-produced technical documentation for the data community, this PR creates a data community section in the GOV.UK developer docs. The first two pieces of documentation are:

- View or extract feedback data in BigQuery
- Zendesk and SmartSurvey feedback data pipeline

### CODE REVIEW

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (mammykins) [inline source/manual/view-extract-feedback-data-bigquery.html.md.erb:14]: how to get access to BigQuery data sets? ask data labs

File: source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb
Comment (mammykins) [inline source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb:14]: can we link to our Google Slides on here, what's the etiquette / rules on linking to files on our drive, given this is public?

**Review by mammykins — COMMENTED** [review]
This makes sense to me.

I think it prudent to get someone who is less familiar with the domain to review and approve, that way we can assess if it's understandable to someone without prior knowledge. I'll create a card for someone.

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (jonathanglassman) [reply source/manual/view-extract-feedback-data-bigquery.html.md.erb:14]: will update

File: source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb
Comment (jonathanglassman) [reply source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb:14]: @mammykins we can link to a file on the drive as long as we say that it requires access and how to request that access. what is the link and how can a user ask for access to that link?

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (mammykins) [inline source/manual/view-extract-feedback-data-bigquery.html.md.erb:40]: the project name is `govuk-bigquery-analytics` not `gov uk-bigquery-analytics`

File: source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb
Comment (mammykins) [reply source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb:14]: Changed my mind, this is sufficient

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (mammykins) [inline source/manual/view-extract-feedback-data-bigquery.html.md.erb:47]: When I copy and paste this in I'm getting an error

`Syntax error: Illegal input character "\342" at [3:13]`, it doesn't like the single quotes around Ticket. Is that just me, or does it happen to you also?

Comment (mammykins) [reply source/manual/view-extract-feedback-data-bigquery.html.md.erb:47]: The same applies to all the single quotes, you'll need to replace them?

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (mammykins) [inline source/manual/view-extract-feedback-data-bigquery.html.md.erb:90]: this is helpful

File: source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb
Comment (mammykins) [inline source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb:85]: TBC? to be confirmed?

**Review by ESKYoung — CHANGES_REQUESTED** [review]
Looks good @jonathanglassman! Not reviewed the tables in `source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb` in detail.

In addition to @mammykins's comments, see mine below. Also wanted to add a comment about this file, but GitHub isn't letting me! Think there's a stray double quotes in this:

    ```
    PERSON_NAME,
    EMAIL_ADDRESS",
    DATE_OF_BIRTH,
    STREET_ADDRESS
    ```

**Review by ESKYoung — DISMISSED** [review]
Looks good @jonathanglassman! Not reviewed the tables in `source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb` in detail. Also wanted to add a comment about this file, but GitHub isn't letting me! Think there's a stray double quotes for `EMAIL_ADDRESS` in this:

    ```
    PERSON_NAME,
    EMAIL_ADDRESS",
    DATE_OF_BIRTH,
    STREET_ADDRESS
    ```

File: source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb
Comment (jonathanglassman) [reply source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb:85]: correct - when I drafted this content with Ian, we didn't have descriptions of all the data fields. Does this information exist anywhere?

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (jonathanglassman) [reply source/manual/view-extract-feedback-data-bigquery.html.md.erb:107]: thanks, will change

File: source/manual/view-extract-feedback-data-bigquery.html.md.erb
Comment (jonathanglassman) [reply source/manual/view-extract-feedback-data-bigquery.html.md.erb:109]: thanks, will change

**Comment by jonathanglassman** [comment]
> Looks good @jonathanglassman! Not reviewed the tables in `source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb` in detail. Also wanted to add a comment about this file, but GitHub isn't letting me! Think there's a stray double quotes for `EMAIL_ADDRESS` in this:
> 
> ```
> ```
> PERSON_NAME,
> EMAIL_ADDRESS",
> DATE_OF_BIRTH,
> STREET_ADDRESS
> ```
> ```

Thanks @ESKYoung I've made changes where I can and will check your unresolved feedback when I get back from annual leave.

**Comment by jonathanglassman** [comment]
@ESKYoung I've addressed your feedback with @st-alice and asked for her review. Once she approves, I think we're good to go.

**Review by st-alice — COMMENTED** [review]
Hi John, I've double checked the SQL code again and can confirm it works, and the Data Studio instructions are clearer now.

I noticed that all the numbered points start with '1'  - not sure if that's what is expected

**Comment by jonathanglassman** [comment]
> Hi @jonathanglassman, I've double checked the SQL code again and can confirm it works, and the Data Studio instructions are clearer now.
> 
> I noticed that all the numbered points start with '1' - not sure if that's what is expected

@st-alice thanks for checking. Yes, the numbered points will render as correctly sequential bullet point

**Review by st-alice — APPROVED** [review]
Thanks for confirming for me - approved my end!

**Pull request merged**
